midp/sensor.js

```diff
@@ -249,9 +250,10 @@ function(addr, sensorsNumber, number, modelAddr) {
     var n = c.mrangeArray.length;
     var arrayAddr = J2ME.newArray(J2ME.PrimitiveClassInfo.J.klass, n);
     var array = J2ME.getArrayFromAddr(arrayAddr);
+    var i32array = new Int32Array(array.buffer, array.byteOffset, array.length * 2);
```

Comment:
This is unrelated to the PR, but I think the current code on intex had a bug so I fixed it while I was here.